pkg/operator/bootstrap.go

```diff
@@ -105,9 +105,6 @@ func (c *Controller) createPVC(accessMode corev1.PersistentVolumeAccessMode, cla
 		return err
 	}
 
-	// "standard" is the default StorageClass name, that was provisioned by the cloud provider
```

Comment:
true for ovirt. you need to switch case our platform

Reply:
We talked offline but adding it here for reference:
the switch case is[1] and it means that only openstack and ovirt will get a PVC created since we both have a default storage class we should be ok

[1] https://github.com/openshift/cluster-image-registry-operator/blob/master/pkg/storage/storage.go#L157-L162